admin_guide/disabling_features.adoc

```diff
@@ -0,0 +1,276 @@
+    feature-gates:
+    - HugePages=false
+----
+
+. Restart the {product-title} master service to apply the changes.
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master-api origin-master-controllers
+----
+endif::[]
+
+
+[[admin-guide-disable-feature-cluster]]
+== Disabling Features for a Node
+
+To turn off a feature for the entire cluster, edit the *_node-config.yaml_*, by default _/etc/origin/node/node-config.yaml_:
+
+. For the feature you want to turn off, enter: `<feature_name>=false` under`proxyArguments`.
++
+For example:
++
+[source,yaml]
+----
+proxyArguments:
+  feature-gates:
+  - HugePages=false
+----
+
+. Restart the {product-title} service for the changes to take effect:
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-node
+----
+endif::[]
+
+To re-enable a disabled feature, edit the master and node configuration files to remove the `<feature_name>=false`
+and restart the master and node services.
+
+==== List of Feature Gates
+
+Use the following list to determine the name of the feature you want to disable: 
+
+[options="header"]
+|===
+| Feature gate| Description
+
+| *Accelerators* 
+| Enables link:https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/[Nvidia GPU] support when using Docker.
+
+| *AdvancedAuditing* 
+| Enables xref:../install_config/master_node_configuration.adoc#master-node-config-advanced-audit[Advanced
+Audit].
+
+| *APIListChunking*
+|Enables the API clients to retrieve LIST or GET resources from API server in chunks.
+
+| *APIResponseCompression*
+| Enables the compression of API responses for LIST or GET requests.
+
+| *AppArmor*
+| Enables AppArmor-based mandatory access control on Linux nodes when using Docker. For more information, see the link:https://kubernetes.io/docs/tutorials/clusters/apparmor/[Kubernetes AppArmor documentation].
+
+| *BlockVolume*
+| Enables the definition and consumption of raw block devices in pods. For more information, see 
+the link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#raw-block-volume-support[Kubernetes Raw Block Volume Support].
+
+| *CPUManager*
+| Enables container-level CPU affinity support. For more information, 
+see xref:../scaling_performance/using_cpu_manager.adoc#scaling-performance-using-cpu-manager[Using CPU Manager].
+
+| *CRIContainerLogRotation*
+| Enables container log rotation for the CRI container runtime.
+
+| *CSIPersistentVolume*
+| Enables discovering and mounting volumes provisioned through a CSI (Container Storage Interface) compatible volume plugin. For more information,
+see the link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md[CSI Volume Plugins in Kubernetes Design Doc].
```

Comment:
CSI Volume Plug-ins in Kubernetes Design Documentation  (I see the actual doc has the title you use,  but we should apply our standards to it anyway)